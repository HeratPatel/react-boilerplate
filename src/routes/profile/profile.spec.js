import React from 'react';
import { shallow } from 'enzyme';
import Profile from './profile';

describe('<Profile />', () => {
  it('Rendering', () => {
    const wrapper = shallow(<Profile />);
    expect(wrapper).toMatchSnapshot();
  });
});
