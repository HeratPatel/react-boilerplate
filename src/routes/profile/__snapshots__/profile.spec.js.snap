// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Profile /> Rendering 1`] = `
<section>
  <h1>
    Profile Page
  </h1>
</section>
`;
